### PR TITLE
Adding Standard_Microsoft sku in CDN pricing tier

### PR DESCRIPTION
**Pull request by hytao**
## Description

Regenerating CDN SDK based on latest swagger spec.
https://github.com/Azure/azure-rest-api-specs/pull/3628

---

This checklist is used to make sure that common guidelines for a pull request are followed.
- [ x] https://github.com/Azure/azure-rest-api-specs/pull/3628
- [ x] **I have read the [contribution guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md).**
- [ x] **The pull request does not introduce [breaking changes](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/Documentation/breaking-changes.md).**

### [General Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#general-guidelines)
- [x ] Title of the pull request is clear and informative.
- [ x] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For more information on cleaning up the commits in your PR, [see this page](https://github.com/Azure/azure-powershell/blob/dev/documentation/cleaning-up-commits.md).

### [Testing Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#testing-guidelines)
- [ x] Pull request includes test coverage for the included changes.

### [SDK Generation Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#sdk-generation-guidelines)
- [x ] If an SDK is being regenerated based on a new swagger spec, a link to the pull request containing these swagger spec changes has been included above.
- [ x] The generate.cmd file for the SDK has been updated with the version of AutoRest, as well as the commitid of your swagger spec or link to the swagger spec, used to generate the code.
- [ x] The `*.csproj` and `AssemblyInfo.cs` files have been updated with the new version of the SDK.

### CODE REVIEW

File: src/SDKs/Cdn/Management.Cdn/Microsoft.Azure.Management.Cdn.csproj [hunk 1]
```diff
@@ -5,7 +5,7 @@
 
 	<PropertyGroup>
 		<Description>Microsoft Azure CDN Management Library</Description>
-		<Version>4.0.1-preview</Version>
+		<Version>4.0.2-preview</Version>
 		<AssemblyName>Microsoft.Azure.Management.Cdn</AssemblyName>    
 		<PackageId>Microsoft.Azure.Management.Cdn</PackageId>
 		<PackageTags>Microsoft Azure CDN management;CDN;Cdn management;</PackageTags>
```
Comment (dsgouda) [inline src/SDKs/Cdn/Management.Cdn/Microsoft.Azure.Management.Cdn.csproj]: Please update the `PackageReleaseNotes` here

**Review by dsgouda — CHANGES_REQUESTED** [review]
Left a couple of comments

File: src/SDKs/Cdn/Management.Cdn/Microsoft.Azure.Management.Cdn.csproj (hunk 1 above)
Comment (hytao) [reply src/SDKs/Cdn/Management.Cdn/Microsoft.Azure.Management.Cdn.csproj]: Release Notes updated

File: src/SDKs/Cdn/Management.Cdn/Properties/AssemblyInfo.cs [hunk 2]
```diff
@@ -8,8 +8,8 @@
 [assembly: AssemblyTitle("Microsoft Azure CDN Management Library")]
 [assembly: AssemblyDescription("Provides management functionality for Microsoft Azure CDN.")]
 
-[assembly: AssemblyVersion("4.0.0.0")]
-[assembly: AssemblyFileVersion("4.0.1.0")]
+[assembly: AssemblyVersion("4.0.1.0")]
```
[the remark this answers is not in the document]
Comment (hytao) [reply src/SDKs/Cdn/Management.Cdn/Properties/AssemblyInfo.cs]: Changes reverted

**Review by dsgouda — CHANGES_REQUESTED** [review]
Was this code generated using generate.ps1? 
Changes to a .txt log file must also be generated please commit those changes

**Comment by dsgouda** [comment]
@azuresdkci Retest this please

**Comment by hytao** [comment]
Logs added. 
Thanks

**Review by dsgouda — APPROVED** [review]
Everything looks great, please squash commits into a single commit and we should be good to merge

**Comment by hytao** [comment]
@dsgouda I'm not sure how to do this on GitHub. Could you help?

**Comment by dsgouda** [comment]
The easiest way I can think of is:
1. Rename your local branch to something else and copy over all the files you changed here to a temp directory
2. Checkout psSdkJson6 on your local repo, pull down latest upstream changes and create a new branch with the same name as this, copy over the files from temp directory here, commit the changes and force push here

**Comment by hytao** [comment]
Hi, @dsgouda , It appears that I was working on branch psSdkJson6 and I am not sure how to make it clean again.
Do you think it is a good idea to restart from the very beginning and create a new PR?

**Comment by dsgouda** [comment]
That would be the safest bet. Please make sure you save your changes somewhere while doing so

**Comment by hytao** [comment]
@dsgouda, I created a new PR https://github.com/Azure/azure-sdk-for-net/pull/4730.

Thanks

**Pull request closed without merging**